pkgs/development/python-modules/cPyparsing/default.nix

```diff
@@ -0,0 +1,20 @@
+{lib, buildPythonPackage, fetchPypi, cython}:
+
+buildPythonPackage rec {
+  pname = "cPyparsing";
+  version = "2.4.5.0.1.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1kcr8v9yx9whbdn7c7rnfg4b6bk3agpa1wblhr2y6n93r8a4bxwy";
+  };
+
+  buildInputs = [ cython ];
```

Comment:
Is this required at runtime? If so use propagatedBuildInputs.

Reply:
Cython isn't, this builds some native extension that is used and propagated by coconut.